app/controllers/events_controller.rb

```diff
@@ -36,6 +36,16 @@ def invalid_request(message = 'invalid request', type: 'invalid_request')
     render json: { error: message, type: type }, status: 400
   end
 
+  def format_integration_error(exception)
+    exception_info = {
+      error: exception.message,
+      type: exception.type,
+      failed_response_body: exception.response_body,
+      failed_response_code: exception.response_code,
+    }
+    render json: exception_info, status: 400
+  end
```

Comment:
do we still need `invalid_request?`? Seems like we should pick one method and use it for everything.